nightly-playground/resources/security-config/internal_users.yml

```diff
@@ -8,31 +8,19 @@ _meta:
 # Define your internal users here
```

Comment:
We can completely remove this file right @gaiksaya ?

Reply:
tests are failing if I remove it. Also wanted to keep a sample for users to view/